snowflake/README.md

```diff
@@ -117,3 +141,82 @@ Snowflake credentials are required.
 
 We are in the process of adding support for applications to Snowflake-powered
 Theia, starting with NetworkPolicy recommendation.
+
+## Network flow visibility with Grafana
+
+We use Grafana as the tool to query data from Snowflake, and visualize the
+networking flows in the cluster(s).
+
+### Configure datasource
+
+Datasource and pre-built dashboards will be provisioned to Grafana. You need to
+configure the datasource in
+`theia/snowflake/grafana/provisioning/datasources/datasource_provider.yaml`.
+
+Available configuration fields are as follow:
+
+ Name                     | Description
+------------------------- | ------------
+ role (Optional)          | Specifies the default access control role to use in the Snowflake session initiated by Grafana.
+ warehouse                | Specifies the virtual warehouse to use once connected.
+ database                 | Specifies the default database to use once connected.
+
+Database name can be found in previous output of the `onboard` command.
```

Comment:
can be found in the output...

and add a link to the relevant subsection above